moirae/extractors/ecm.py

```diff
@@ -7,7 +9,7 @@
 from scipy.optimize import differential_evolution
 from battdat.data import BatteryDataset
 from battdat.postprocess.integral import CapacityPerCycle, StateOfCharge
-from typing import Tuple
+from battdat.postprocess.tagging import AddSteps, AddState

```

Comment:
Thanks for moving the standard module to the top of the imports. Warms my pedantic heart!